javascript/index.js

```diff
@@ -14,52 +14,91 @@ let milUni = document.getElementById('milUni');
 let splits = document.getElementById('splits');
 
 function printTime() {
-  // ... your code goes here
-}
 
+  printMinutes();
+  printSeconds();
+
+}
 function printMinutes() {
-  // ... your code goes here
+  const mins = chronometer.twoDigitsNumber(chronometer.getMinutes());
+    minDec.innerText = mins[0];
+    minUni.innerHTML = mins[1];
 }
 
 function printSeconds() {
-  // ... your code goes here
+  const secs = chronometer.twoDigitsNumber(chronometer.getSeconds());
+  secDec.innerText = secs[0];
+  secUni.innerHTML = secs[1];
 }
 
 // ==> BONUS
 function printMilliseconds() {
   // ... your code goes here
 }
 
-function printSplit() {
-  // ... your code goes here
-}
+  function printSplit() {    
+    let splitsCount = document.getElementById('splits')  
```

Comment:
You already have `let splits` on line 14 so this is unnecessary, you can replace line 43 with `splitsCount.appendChild(newSplit)`